api/role/schema.py

```diff
@@ -0,0 +1,38 @@
+import graphene
+from graphene_sqlalchemy import ( 
+    SQLAlchemyObjectType, 
+    SQLAlchemyConnectionField
```

Comment:
I think *SQLAlchemyConnectionField* since we are no longer using relay interfaces

Reply:
🤭Oops, good catch! I'll fix it.